docs/pages/includes/role-spec.mdx

```diff
@@ -120,17 +120,19 @@ spec:
     windows_desktop_logins: [Administrator, '{{internal.logins}}']
 
     # node_labels: a user with this role will be allowed to connect to
-    # SSH nodes, which labels match expressions below.
+    # SSH nodes with labels matching below.
     node_labels:
       # literal strings:
       'env': 'test'
       # the wildcard ('*') means "any node"
       '*': '*'
       # a list of alternative options:
       'region': ['us-west-1', 'eu-central-1']
-      # regular expressions start with ^ and end with $
-      # Teleport uses golang regexp syntax.
-      # of the list example above can be expressed as:
+      # Regular expressions start with ^ and end with $.
+      # Teleport uses Go's regular expression syntax:
+      # https://github.com/google/re2/wiki/Syntax
+      # The list example above can be expressed as:
+      # 'region': '^us-west-1|eu-central-1$'
```

Comment:
I copied this text from the existing comment on `app_labels`, it seems much better